storage/src/main/java/org/apache/uniffle/storage/handler/impl/PooledHdfsShuffleWriteHandler.java

```diff
@@ -55,7 +55,7 @@ public PooledHdfsShuffleWriteHandler(
     // todo: support init lazily
     try {
       for (int i = 0; i < maxConcurrency; i++) {
-        queue.offer(
```

Comment:
Although it should always be successful here, we should check the result of `BlockingQueue#offer()`.

Reply:
Yes, it is possible to add `@SuppressFBWarnings` annotation.
But I think the original code looks strange, should we do some refactor?